Comparing changes against meejah/2916.grid manager proposal.5

edison12a opened the pull request.

meejah commented:
So is [860d835](https://github.com/meejah/tahoe-lafs/pull/3/commits/860d835eeb25f638dd0821904ddf41f0af65ec65) a merge commit? (or, can you say how you produced that?)

edison12a commented:
The biggest part of [860d835](https://github.com/meejah/tahoe-lafs/pull/3/commits/860d835eeb25f638dd0821904ddf41f0af65ec65) is a merge commit.
The rest comes from fixing conflicts and bugs that resulted from the merge until I was able to run the test suite.

meejah commented:
That's going to be ... extremely hard to review. That is, how can I tell which of those 16 thousands lines are "just merge" and which are "changes you made"...?

edison12a commented:
The commits are separated. I made the merge commit first and I’m making my changes in separate commits.
Will that be an adequate solution?

(Had replied this via email but I guess it didn't automatically get forwarded)

edison12a commented:
Status Update: Still fixing tests. Currently running allmydata.test.cli.test_invite.

meejah commented:
> The commits are separated. I made the merge commit first and I’m making my changes in separate commits.
> Will that be an adequate solution?

Yeah, if you can push that I can take a look ... sounds better :) 
(I still just see the one big commit here)

meejah commented:
Bump; can you push the version with separated commits please?

meejah commented:
bump; I still only see a single commit with "everything", if you can push the individual commits instead that might make it possible to read :) thanks

edison12a commented:
Status: 
Hi my tests status is PASSED (skips=19, successes=1737) but I previosly disables 2 test classes. 
I'm still working to separate the changes into individual commits with varying version numbers.

edison12a commented:
Earlier on, Had an issue with file systems. My git diff had a lot of empty files because I had moved the files between Operating systems during a local backup. I fixed that issue. 

I'm Currently dealing with organizing the changes in order. This has taken longer than anticipated due to a local repos mixup that is my fault.

edison12a commented:
@meejah I'm working to get this into a reviewable state.

meejah commented:
Who controls the `asyncverse` account?

edison12a commented:
Hi @meejah, I own the @asyncverse account. I can find ways of removing it if its causing any confusion.

meejah commented:
okay, that's fine ownership-wise .. that _is_ the commit that needs to be re-done though (I assume) since it seems to be a bunch more than just merge..?
